packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.styles.ts

```diff
@@ -150,6 +150,21 @@ export const getMenuItemStyles = memoizeFunction((
       verticalAlign: 'middle',
       flexShrink: '0',
       fontSize: FontSizes.mini
+    },
+    splitDivider: {
+      backgroundColor: ContextualMenuDividerColor,
+      position: 'absolute',
+      width: 1,
```

Comment:
You shouldn't have to apply a top and bottom dimension if you are using flex to center align it. A height and width should suffice.